tests/phpunit/Configuration/ConfigurationFactoryTest.php

```diff
@@ -143,7 +144,8 @@ public function test_it_can_create_a_configuration(
         bool $expectedShowMutations,
         ?float $expectedMinCoveredMsi,
         array $expectedIgnoreSourceCodeMutatorsMap,
-        bool $inputExecuteOnlyCoveringTestCases
+        bool $inputExecuteOnlyCoveringTestCases,
+        ?string $mapSourceClassToTest,
```

Comment:
I'm happy with the current change, just mentioning this in case you didn't think of it and you would prefer it:

```suggestion
        ?string $mapSourceClassToTest = MapSourceClassToTestStrategy::SIMPLE,
```

It would allow you to skip the modification of all the provide data sets